app/konnect/service-catalog/integrations/gitlab.md

```diff
@@ -0,0 +1,66 @@
+---
+title: GitLab Integration
+content-type: reference
+beta: true
+
+discovery_support: true
+discovery_default: true
+bindable_entities: "Projects"
+mechanism: "pull/ingestion model"
+---
+
+_Type: External_
+
+The GitLab integration allows you to associate your {{site.service_catalog_name}} service to one or more [GitLab projects](https://docs.gitlab.com/ee/user/get_started/get_started_projects.html). 
+
+For each linked Project, the UI can show a **Project Summary** with simple data pulled from the GitLab API, such as the number of open issues, open merge requests, contributors, languages, and latest releases.
+
+## Prerequisites
+
+* You need the [Owner GitLab role](https://docs.gitlab.com/ee/user/permissions.html) to authorize the integration. This is required for event ingestion.
+* Only [GitLab.com subscriptions](https://docs.gitlab.com/ee/subscriptions/gitlab_com/) are supported at this time
+
+## Authorize the GitLab integration
+
+1. From the **{% konnect_icon servicehub %} Service Catalog** in {{site.konnect_product_name}}, select **[Integrations](https://cloud.konghq.com/us/service-catalog/integrations)**. 
+2. Select **GitLab**, then **Install GitLab**.
+3. Select **Authorize**. 
+
+## View GitLab specs in Service Catalog
+
+You can map specs to a Service Catalog service and view them in {{site.konnect_short_name}}.
+
+1. From the **{% konnect_icon servicehub %} Service Catalog** in {{site.konnect_product_name}}, select **[Services](https://cloud.konghq.com/us/service-catalog)**. 
+2. Select a service, then open the **API Specs** tab.
+2. Select **Add API Spec**.
+3. Choose GitLab as the **Source**, then pick your spec and name it.
+
+Once it's uploaded, you can view the rendered spec on the API Specs tab in structured (UI), YAML, or JSON format, and download it in either YAML or JSON format.
+
+## Resources
+
+Entity  | Description
+-------|-------------
+Projects | Organizes all the data for a specific development project that relates to a Service Catalog Service.
+
+## Events
+
+This integration supports events.
+
+You can view the following event types for linked projects from the {{site.konnect_product_name}} UI:
+
+* Opened merge requests
+* Merged merge requests
+
+
+## Discovery information
```

Comment:
suggestion: there's info here that's not user-facing and some that's no longer relevant (cc @miko-bautista)

![Screenshot 2025-01-16 at 11 32 19 AM](https://github.com/user-attachments/assets/a8c0792f-9735-4933-a4f6-1164f58eb592)

- There's no concept of "enabled" for integration discovery in KSC
- `Bindable entities` is close to `binding` which is a term we're trying to move away from externally. Instead, use "Resources" and/or "Resource Types"
- "Discovery mechanism" is not a user-facing detail

I think this section can be as concise as:

> GitLab Projects will be automatically discovered once the integration is authorized.

---

Rest of the docs look great! 🙂

Reply:
@Drew-Kimberly Thanks for the feedback! Since this is reusable content that is used by multiple integrations, I removed the 1st and 4th bullets, and reworded the remaining two per your suggestions.